packages/recommend-react/src/useFrequentlyBoughtTogether.ts

```diff
@@ -103,7 +105,7 @@ export function useFrequentlyBoughtTogether<TObject>({
   ]);
 
   return {
-    ...result,
+    recommendations,
```

Comment:
Why did we change all these returns? The reasoning was that if the API adds information in the response, we don't need to release another version of the library to leverage them.

Reply:
With Batching the typing becomes kind of chaotic around here, i'm open to suggestions

Reply:
Fixed in latest commits